src/Core/Package.vala

```diff
@@ -297,6 +297,23 @@ public class AppCenterCore.Package : Object {
         }
     }
 
+    private string? _author_id = null;
+    public string? author_id {
+        get {
+            if (_author_id != null) {
+                return _author_id;
+            }
+
+            _author = component.get_developer ().get_id ();
+
+            if (_author_id == null) {
+                return _author;
+            }
+
+            return _author_id;
+        }
```

Comment:
This fallback means the `author_id` property never returns null, but you check for null in AuthorView. Maybe we shouldn't fallback to `_author` here?